package.json

```diff
@@ -40,7 +40,7 @@
     "@babel/preset-react": "^7.0.0-rc.3",
     "babel-core": "^7.0.0-bridge.0",
     "babel-eslint": "^9.0.0-beta.3",
-    "babel-jest": "^23.4.2",
+    "babel-jest": "^23.6.0",
     "babel-loader": "^8.0.0-beta.6",
     "babel-watch": "git://github.com/hedgepigdaniel/babel-watch.git",
```

Comment:
hows ur PR going to the main project. can i try and help push it through and merged?

Reply:
tbh I haven't made a PR - this branch is a combination of about 4 open PRs from others and a fix or two I made myself (the main problem was CTRL-C created a 100% CPU hung process). Last I checked the package was looking for a new maintainer - if I had a bit more spare time I might volunteer.

Reply:
Ahh a shame, I get it. Good idea to put your hand up anyways. Even if there’s delays- at least it’s yours to fix when you need to.  Or farm out. 

Btw the guy is giving us respond name on npm. Transfer tomorrow so we have a clean name